### DISCUSSION TITLE
background ячейки

### DISCUSSION THREAD

**Discussion by dmbrtlt** [totumonline/totum-ru-issues-and-discussions#623] [Как это сделать?]
Почему не раскрашивается вся кнопка?
![image](https://github.com/user-attachments/assets/115852b9-4c1b-46d2-8dd9-04517a155aae)
![image](https://github.com/user-attachments/assets/adc452f2-ec3f-44ab-af07-97d71a3a111e)

**Comment by erpic-ru** [comment]
[В настройках секции](https://ru.docs.totum.online/interface#sx-section-assignment) нужно поставить border: false

**Reply by dmbrtlt** [reply]
Спасибо!
